hibernate-testing/src/main/java/org/hibernate/testing/junit4/BaseNonConfigCoreFunctionalTestCase.java

```diff
@@ -450,6 +451,11 @@ protected void prepareTest() throws Exception {
 
 	@After
 	public final void afterTest() throws Exception {
+		// see https://github.com/hibernate/hibernate-orm/pull/3412#issuecomment-678338398
+		if ( getDialect() instanceof H2Dialect ) {
+			ReflectHelper.getMethod( Class.forName( "org.h2.util.DateTimeUtils" ), "resetCalendar" ).invoke( null );
```

Comment:
I might be wrong, but it seems we can invoke the public util method directly:
```
DateTimeUtils.resetCalendar();
```
No?

Reply:
We could, but in #3412 some people voted for reflective invocation so that no dependency is needed here.